test/gtest/uct/test_uct_perf.cc

```diff
@@ -153,18 +153,22 @@ UCS_TEST_P(test_uct_perf, envelope) {
 
     if (has_transport("tcp")) {
         check_perf = false; /* TODO calibrate expected performance based on transport */
+        max_iter   = 100000lu;
```

Comment:
maybe reduce it even more (say 1000), since we do not check perf?

Reply:
UCT: 4095 ms -> 61 ms
UCP:  -> 1150 ms

I have no objections, @yosefe ?

then we can remove `max_iter` variable and do:
```
...
    if (!check_perf) {
        test.iters = 1000lu;
    }
    run_test(test, 0, check_perf, "", "");
}
...
```